### PR TITLE
Add codex/codex-icons/codex-design-tokens w/ npm auto-update

### PR DESCRIPTION
**Pull request by lucaswerkmeister**
[Codex][1] is the design system for Wikimedia. Having it available on cdnjs will be useful for developers of Wikimedia-related tools who want to match the design of Wikimedia sites (see also [T338834][2]).

The unscoped [`codex` npm package][3] has not been updated in over ten years (and as “a simple tool for building static [websites]” probably would not be very useful on cdnjs anyways), so I hope nobody will mind using the unprefixed `codex` name directly for `@wikimedia/codex`. (There seems to be some precedent for this among existing npmjs packages too, such as `angular` for `@angular/core` or `treejs` for `@widgetjs/tree`.)

[1]: https://www.mediawiki.org/wiki/Special:MyLanguage/Codex
[2]: https://phabricator.wikimedia.org/T338834
[3]: https://www.npmjs.com/package/codex

### CODE REVIEW

**Comment by lucaswerkmeister** [comment]
For convenience, checking against the [popularity policy](https://github.com/cdnjs/packages/blob/master/CONTRIBUTING.md#policy-rules-and-guidelines):

- [`@wikimedia/codex`](https://www.npmjs.com/package/@wikimedia/codex): 589 + 584 + 450 + 730 = **2353** downloads the past four weeks, well over the threshold
- [`@wikimedia/codex-icons`](https://www.npmjs.com/package/@wikimedia/codex-icons): 238 + 260 + 245 + 289 = **1032** downloads the past four weeks, closer but still over the threshold
- [`@wikimedia/codex-design-tokens`](https://www.npmjs.com/package/@wikimedia/codex-design-tokens): 262 + 226 + 216 + 174 = **878** downloads the past four weeks, barely above (but I hope you’d accept this one anyways as part of Codex as a whole)

The unscoped [`codex`](https://www.npmjs.com/package/codex), whose name is being “taken over” here, has 6 + 39 + 40 + 25 = 110 downloads the past four weeks (and 64 stars [on GitHub](https://github.com/logicalparadox/codex)), so it would not be eligible.

File: packages/c/codex-icons.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "front-end"
+  ],
+  "autoupdate": {
+    "source": "npm",
+    "target": "@wikimedia/codex-icons",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*.@(js|mjs|json|svg)"
```
Comment (MattIPv4) [inline packages/c/codex-icons.json:18]:
```suggestion
          "*.@(js|mjs|json)",
          "images/*.svg"
```
It looks like the actual icons are in a sub-directory?

Comment (lucaswerkmeister) [reply packages/c/codex-icons.json]: Good catch, thanks – I guess I wasn’t sure if the blobs are directory-recursive or not.

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Comment by lucaswerkmeister** [comment]
Thanks! [codex](https://cdnjs.com/libraries/codex) is working fine, but [codex-icons](https://cdnjs.com/libraries/codex-icons) and [codex-design-tokens](https://cdnjs.com/libraries/codex-design-tokens) 404 and I don’t know why… do libraries without JS files automatically get filtered out, perhaps?

**Comment by MattIPv4** [comment]
It looks like they haven't been processed yet: https://github.com/cdnjs/logs/commits/prod

**Comment by lucaswerkmeister** [comment]
I see, thanks.

**Comment by lucaswerkmeister** [comment]
You’re right, now they’re both there. Thanks again!

**Pull request merged**